KenanLab1.java

```diff
@@ -1,15 +1,21 @@
 import java.awt.event.*;
+import java.util.*;
 
 public class KenanLab1 implements KeyListener
 {
+private int[][] maze;
+private int xLoc, yLoc;
    public static void main (String[] args)

```

Comment:
spacing matters. I will cover this in review, but you will need to keep your code clean.